js-opentracing/index.js

```diff
@@ -0,0 +1,36 @@
+const {
+  lightstep,
+  opentelemetry,
+} = require('lightstep-opentelemetry-launcher-node');
+const opentracing = require('opentracing');
+const axios = require('axios');
+
+// TraceShim is needed to join OpenTelemetry with OpenTracing
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+
+const sdk = lightstep.configureOpenTelemetry({
+  accessToken: 'YOUR_TOKEN',
+  serviceName: 'CollectorTraceExporter with OpenTracing',
```

Comment:
can you call this js-otshim-example' for consistency with how the other services in these examples are named?